src/components/CaptureRoutes/CaptureRoutes.tsx

```diff
@@ -1,11 +1,11 @@
 import type { JSX } from "react";
 import { useEffect } from "react";
-import { Outlet, useLocation } from "react-router";
+import { useLocation } from "react-router";
 
 import useAnalytics from "hooks/useAnalytics";
 import useFeatureFlags from "hooks/useFeatureFlags";
 
-const CaptureRoutes = (): JSX.Element => {
+const CaptureRoutes = (): JSX.Element | null => {
```

Comment:
```suggestion
const CaptureRoutes = (): null => {
```

Should we have the signature reflect that it doesn't render anything? Or are we leaving the door open for it to render stuff in the future?